app/src/main/java/com/example/customcalculatorv2hw02/CalcFragment.kt

```diff
@@ -0,0 +1,309 @@
+package com.example.customcalculatorv2hw02
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Toast
+import androidx.fragment.app.Fragment
+import com.example.customcalculatorv2hw02.databinding.FragmentCalcBinding
+import com.example.customcalculatorv2hw02.engine.StringExpressionAnalyzer
+
+class CalcFragment : Fragment() {
+
+    private var _binding: FragmentCalcBinding? = null
+    private val binding get() = requireNotNull(_binding) {
+        "View was destroyed"
+    }
+
+    val calc = StringExpressionAnalyzer()
```

Comment:
```suggestion
    private val calc = StringExpressionAnalyzer()
```
непонятно с неймингом, вроде класс анализатор, а поле calc